modules/mesaides/templates/nginx_config.erb

```diff
@@ -2,7 +2,11 @@
 # Modifications should be made in that template
 
 server {
+<% if @is_default -%>
   listen              *:80 default_server;
+<% else -%>
+  listen              *:80;
+<% end -%>
```

Comment:
> Maybe adding `@is_default` inline would be better, in case we end up putting more stuff than `*:80` in `listen`, it will be clear `@is_default` only impacts `default_server`?